### PR TITLE
Add a README to specify test expectations

### PR DESCRIPTION
**Pull request by aabrook**
Bringing in, and updating, the description of the repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+**Scheduled Performance Input**
+
+```
+band: String 
+start: DateTime 
+finish: DateTime 
+```
+
+Please provide a link to a private github repository containing your solution and instructions how to launch.
```
Comment (nedmules-sked) [inline README.md]: To allow us to use the verifier, the instructions will need to be changed back to the original that requests a build script and a run script to be provided:

```
Please provide an archive file containing your source code, plus two simple scripts in the root directory:

a. build.sh or build.bat - builds the program
b. run.sh or run.bat - a wrapper that runs the program - This script takes a single argument as described above and executes the program.

We will check your solution by building it using the ​build​ script and then executing the ​run​ script multiple times with various input files.
```

**Comment by nedmules-sked** [comment]
Note that this repo is for the verifier script for the tech test. This file describes the verifier and should really be moved/merged to README.md: https://github.com/Skedulo/backend-tech-test/blob/master/backend/music-schedule/verifier/Readme.txt


Aside from that, I think it would be best to have a single source of truth for the contents of the tech test question so there is no confusion around which version to send out or which version someone used for their solution.

It might be better just to link to the source of truth from here (Google Docs) instead of copying the content.

File: README.md (hunk 1 above)
Comment (aabrook) [reply README.md]: I'll bring that in as that makes it easy and what I would like for our candidates to do.
The best thing that people liked about hackerrank was working towards a known goal, so working towards this codebase gives them that confidence

**Comment by aabrook** [comment]
> Note that this repo is for the verifier script for the tech test. This file describes the verifier and should really be moved/merged to README.md: [https://github.com/Skedulo/backend-tech-test/blob/master/backend/music-schedule/verifier/Readme.txt](https://github.com/Skedulo/backend-tech-test/blob/master/backend/music-schedule/verifier/Readme.txt?rgh-link-date=2021-01-06T05%3A54%3A58Z)
> 
> Aside from that, I think it would be best to have a single source of truth for the contents of the tech test question so there is no confusion around which version to send out or which version someone used for their solution.
> 
> It might be better just to link to the source of truth from here (Google Docs) instead of copying the content.

I would like to keep it contained within Github and not in a Google doc so we can track changes and have people fork the repo directly. One of the outcomes from the backend guild was seeing the progress people make when solving problems and git gives us that in spades.

**Pull request merged**